polygon-id/id3/id3.json

```diff
@@ -2407,9 +3732,6 @@
               "type": "string"
             }
           }
-        },
```

Comment:
this id is okay to be removed?

Reply:
i am not seeing any change really here??

Reply:
ok